views/widgets/Right_Text_Slider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/* @var $image string */
+/* @var $instance array */
+?>
+<div class="pb-widget pb-widget-num-0">
+	<div class="pb-widget-inner">
```

Comment:
this class is inserted automatically with $instance['before_widget']

Reply:
removed